mob

```diff
@@ -394,11 +394,7 @@ if args.name:
 image_and_tag = get_image(conf='.mobconf', image=args.image, tag=args.tag)
 if not args.no_pull:
     pull_image_command = ["docker", "image", "pull", image_and_tag]
-
-    if args.verbose:
-        maybe_run(pull_image_command)
-    else:
-        maybe_run_quiet(pull_image_command)
+    maybe_run(pull_image_command)
```

Comment:
@jgreat i changed this because, i had a negative user experience when the docker image was revved. The mob prompt command seems to just hang for several minutes without any output. This way, it shows that output. Since the first time the user uses the tool, it's going to do that, I think this is a big improvement